pages/projects/ios/minspeed/index.tsx

```diff
@@ -20,7 +20,8 @@ export default function MinSpeedSupportPage() {
                     <div className="w-full p-1 md:p-2">
                         <div className="flex flex-wrap w-1/3">
                             <div className="w-full p-1 md:p-12">
-                                <Link href="https://apps.apple.com/us/app/minspeed/id6470914321"><img alt="gallery" className="block object-cover object-center w-full h-full rounded-lg"
+                                {/* replace link below after deployment */}
```

Comment:
Insdead of comment, it should be `https://apps.apple.com/us/app/minspeed/id6470914321`

Reply:
fixed